ui/src/authors/components/AuthorResultItem.jsx

```diff
@@ -21,11 +21,12 @@ class AuthorResultItem extends Component {
     );
     const arxivCategories = metadata.get('arxiv_categories');
     const experiments = metadata.get('project_membership');
+    const nativeName = metadata.getIn(['name', 'native_names', 0]);
```

Comment:
what if there's more than one native name?

Reply:
we only allow 1 native name in the author form. Are there cases with more?

Reply:
the schema has a list of native_names so it allows for more than one to be there. But do we really want to display all native names in the title like that?

Reply:
maybe not in the title, but in the detailed record we probably should.

Reply:
We checked and found only 6 authors out of 125k with more than one native name. After discussion with Stella, we will display only first native name